adapters/smartadserver/smartadservertest/exemplary/native-app.json

```diff
@@ -135,6 +135,7 @@
             "content": {
               "url": "https://play.google.com/store/apps/details?id\u003djp.gocro.smartnews.android",
               "userrating": "4.2",
+              "livestream": 0,
```

Comment:
This was previously excluded from the bid request since it was considered a default value. We've better aligned the OpenRTB with the IAB spec and this field is now included for 0 values. Test updated.